addons/account/models/account_journal_dashboard.py

```diff
@@ -124,23 +124,36 @@ def _get_json_activity_data(self):
 
     def _query_has_sequence_holes(self):
         self.env['account.move'].flush_model(['journal_id', 'date', 'sequence_prefix', 'sequence_number', 'state'])
+        to_check = {}
+        for journal in self:
+            # A branch company is locked when the parent is locked.
+            # Parent companies of the journal company can not add moves to the journal.
+            # Thus it is good enough to consider all moves in the journal after `lock_date`.
+            # It finds all holes that can still be corrected.
+            lock_date = journal.company_id.fiscalyear_lock_date
+            to_check.setdefault(lock_date, []).append(journal.id)
```

Comment:
```suggestion
        to_check = self.grouped(lambda j: j.company_id.fiscalyear_lock_date)
```

(you can keep the comments ofc)

Reply:
done (thanks)